helm-charts/explorer/templates/explorerdb-configmap.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: explorer-db-env
+  namespace: {{ $.Values.namespace }}
+  labels:
+    name: {{ .Release.Name }}-db
+    app.kubernetes.io/name: {{ .Release.Name }}-db
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}  
+  annotations: 
+    {{- toYaml .Values.annotations | nindent 6 }} 
+data:
+  DATABASE_DATABASE: {{ $.Values.db.name | default "fabricexplorer" }}
+  DATABASE_USERNAME: {{ $.Values.db.username | default "hppoc" }}
+  DATABASE_PASSWORD: {{ $.Values.db.password | default "password" }}
```

Comment:
Kubernetes secrets is a better place to load the secret values such as password.